### PR TITLE
Working on e2e test fix - Closes #3456

### LINKED ISSUE #3456: Fix broken e2e tests due to adapting to Lisk SDK v5
### Description

e2e need to be refactored and updated due to api changes in lisk sdk v5.

### Motivation

We would like to test features using e2e tests.

### Acceptance Criteria

- All e2e tests should pass.
- Mock data/constants should be updated.

### PR DESCRIPTION
**Pull request by ikem-legend**
### What was the problem?
This PR resolves #3456

### How was it solved?
By updating the Cypress configuration

### How was it tested?
Manually running end to end tests

### CODE REVIEW

File: test/cypress/features/delegateReg.feature [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-    And I click on votesTab
-    Given I click on becomeDelegateLink
-    Then I see this title: Register delegate
-    When I enter the delegate name
-    And I go to confirmation
-    And I click on confirmButton
-    Then I see successful message
-    When I am on Wallet page
-    Then The latest transaction is Delegate registration
+#   Background:
```
Comment (ManuGowda) [inline test/cypress/features/delegateReg.feature:2]: maybe create an issue to fix them, if not fixed in this PR?

**Review by ManuGowda — APPROVED** [review]
Great job @reyraa 🎉 
For the commented e2e tests, either we can address them here https://github.com/LiskHQ/lisk-desktop/issues/3734
or you can fix it in this PR as well.

But rest LGTM once login tests are green

File: test/cypress/features/common/common.js [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
-import { networks, urls, accounts, ss } from '@constants'
-import { settings } from '@constants';
+import { networks, urls, accounts, ss, settings } from '../../../constants';
 
 const txConfirmationTimeout = 15000;
 
-Given(/^Network switcher is disabled$/, function() {
+Given(/^Network switcher is (enabled|disabled)$/, function (status) {
+  const showNetwork = status === 'enabled';
+  console.log('showNetwork', showNetwork);
```
Comment (ManuGowda) [inline test/cypress/features/common/common.js:9]: remove console.log

**Review by ManuGowda — APPROVED** [review]

File: test/cypress/features/delegateReg.feature (hunk 1 above)
Comment (reyraa) [reply test/cypress/features/delegateReg.feature:2]: There's a ticket for this already. #3734

**Pull request merged**